plugins/DuskVerb/tools/tuner/mdr_earlyfield_sweep.py

```
@@ -0,0 +1,125 @@
+# Repo root = .../plugins/DuskVerb/tools/tuner/<this> -> parents[4]. Override
+# with REPO_PATH for non-standard checkouts.
+REPO = os.environ.get("REPO_PATH", str(Path(__file__).resolve().parents[4]))
+REND = f"{REPO}/build/tests/duskverb_render/duskverb_render"
+ANCH = os.path.expanduser("~/projects/dusk-audio-tools/tuner_runs/anchors")
+FC   = f"{REPO}/plugins/DuskVerb/tools/tuner/full_check.py"
+WET  = ["--param", "Dry/Wet=1.0", "--param", "Bus Mode=1", "--param", "Freeze=0",
+        "--sustained-pink-seconds", "4.0"]
+STIM = ["impulse", "noiseburst", "snare", "sine1k", "sustained"]
+NAME = "Medium Drum Room"
+ADIR, APREF = f"{ANCH}/vvv-fat-snare-room", "vvv-fat-snare-room"
+WORK = "/tmp/dattorro_sweep"
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/dattorro_sweep"

(S108)